pulsar-client/src/main/java/org/apache/pulsar/client/impl/PersistentAcknowledgmentsGroupingTracker.java

```diff
@@ -517,6 +517,13 @@ public void flushAndClean() {
         pendingIndividualAcks.clear();
```

Comment:
Just wondering why the existing `flushAndClean()` method couldn't be used? It seems that `pendingIndividualBatchIndexAcks.clear()` is missing from this method. I wonder if that's an issue in this method?

Reply:
1. We can use the `flushAndClean` method, but it performs some additional unnecessary operations. Therefore, I added the `clean` method to simply reset the `acknowledgmentsGroupingTracker`. If you think this is not appropriate, I can reuse the flushAndClean method.
2. The `flushAndClean` method internally removes elements from `pendingIndividualBatchIndexAcks`, so there is no issue with this method.

Reply:
@lhotari I switched to using the `flushAndClean` method. Could you take another look, please? Thank you very much!